src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/MetadataController.kt

```diff
@@ -37,6 +42,26 @@ class MetadataController(val apiClient: HintrAPIClient,
         return apiClient.getUploadMetadata(id)
     }
 
+    data class TypesPayload(val types: Array<String>)
+
+    @PostMapping("/review-inputs/{iso3}")
+    @ResponseBody
+    fun reviewInputMetadata(@PathVariable("iso3") iso3: String,
+                            @RequestBody req: TypesPayload): ResponseEntity<String>
+    {
+        val files = req.types.map {
+            val fileType = when(it.lowercase()) {
+                "anc" -> FileType.ANC
+                "programme" -> FileType.Programme
+                "survey" -> FileType.Survey
+                "shape" -> FileType.Shape
```

Comment:
yep youre correct, removed shape from frontend request and this controller, and created ticket to remove the filters from the response to validation: https://mrc-ide.myjetbrains.com/youtrack/agiles/103-64/current?issue=mrc-5282

Reply:
I am keeping the shape file in because i found out that the api is using it, here: https://github.com/mrc-ide/hintr/pull/501/files#diff-0e93057f8c1b1860a10852d0c43be9d3d9744ec0893c17ea858068d346ccf773R158